### PR TITLE
Master

### PR DESCRIPTION
**Pull request by puneetsaraswat**

### CODE REVIEW

File: 201-application-gateway-url-path-based-routing/azuredeploy.parameters.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (singhkay) [inline 201-application-gateway-url-path-based-routing/azuredeploy.parameters.json]: @puneetsaraswat Can you update the parameters file to add the schema and parmeters sections? See here https://github.com/Azure/azure-quickstart-templates/blob/master/101-azure-dns-new-zone/azuredeploy.parameters.json

Comment (puneetsaraswat) [inline 201-application-gateway-url-path-based-routing/azuredeploy.parameters.json]: Done.

File: 201-application-gateway-url-path-based-routing/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+        "Standard_Small",
+        "Standard_Medium",
+        "Standard_Large"
+      ],
+      "defaultValue": "Standard_Medium",
+      "metadata": {
+        "description": "Sku Name"
+      }
+    },
+    "capacity": {
```
Comment (singhkay) [inline 201-application-gateway-url-path-based-routing/azuredeploy.json]: Is there a max capacity limit? If so, then consider using the maxValue property on the parameter

File: 201-application-gateway-url-path-based-routing/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+  "variables": {
+    "applicationGatewayName": "applicationGateway1",
+    "publicIPAddressName": "publicIp1",
+    "virtualNetworkName": "virtualNetwork1",
+    "subnetName": "appGatewaySubnet",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "subnetRef": "[concat(variables('vnetID'),'/subnets/',variables('subnetName'))]",
+    "publicIPRef": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPAddressName'))]",
+    "applicationGatewayID": "[resourceId('Microsoft.Network/applicationGateways',variables('applicationGatewayName'))]",
+    "apiVersion": "2015-05-01-preview"
```
Comment (singhkay) [inline 201-application-gateway-url-path-based-routing/azuredeploy.json]: This can be 2015-06-15 for Network resources, right?

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**